### DISCUSSION TITLE
False negative "progress" plug-in yaml.schemas

### DISCUSSION THREAD

**Discussion by rei-vilo** [squidfunk/mkdocs-material#3524] [Q&A] [answered]
Hi!

I have installed [redhat.vscode-yaml](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml) and yaml.schemas as per https://squidfunk.github.io/mkdocs-material/creating-your-site/?h=vscode+yaml#minimal-configuration.

However, a red line is displayed under the `progress` plug-in. 

![Capture 2022-01-31 15-26-13](https://user-images.githubusercontent.com/951776/151811473-5971c4f3-4e64-4271-bf48-82181ba6921f.png)

How to remove this false negative?

Thank you!

**Comment by squidfunk** [comment, marked as answer]
We have no schema for this plugin, so the way to fix it would be to provide one. You're invited to add it here:
https://github.com/squidfunk/mkdocs-material/tree/master/docs/schema/plugins

I can't provide schemas for all plugins, so if a plugin and/or extension is expected to work with Material for MkDocs, users and plugin authors should add it here 😊
